package.json

```diff
@@ -64,6 +64,7 @@
     "@babel/preset-env": "7.22.14",
     "@babel/preset-react": "7.22.5",
     "@testing-library/react": "14.0.0",
+    "@testing-library/jest-dom": "^6.1.4",
```

Comment:
the caret can be removed, we don't use them, I think it's to do with our renovate config but maybe @goapunk knows better?

Reply:
the coveralls small reductions can be a bit mysterious?! this might be adding the mod stuff to the serializer?! although coveralls is not currently loading for me so I can't check, we don't worry until it's more then 0.5 reduction